ddlib/ddlib/util.py

```diff
@@ -98,7 +98,7 @@ def normalize_type_name(ty):
   if ty in CANONICAL_TYPE_BY_NAME:
     return CANONICAL_TYPE_BY_NAME[ty]
   else:
-    for patt,ty_canonical in CANONICAL_TYPE_BY_REGEX.iteritems():
+    for patt,ty_canonical in CANONICAL_TYPE_BY_REGEX.items():
```

Comment:
This dictionary has only six things in it and it's a constant, so the risk of impacting performance is small and could go in either direction. We can definitely find some way to use an iterator in both py2 and py3 here, but it'll make the code more complex. In your experience, is this piece of code a hot path?